src/test/java/subway/section/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,174 @@
+package subway.section;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import subway.station.StationResponse;
+import subway.testsupport.AcceptanceTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static subway.line.LineAcceptanceTest.지하철_노선_Id로_조회됨;
+import static subway.line.LineAcceptanceTest.지하철_노선_생성됨;
+import static subway.station.StationAcceptanceTest.지하철역_등록됨;
+
+@DisplayName("지하철 구간 관련 기능")
+public class SectionAcceptanceTest extends AcceptanceTest {
+
+    private Long lineId;
+    private Long 강남역_Id;
+    private Long 역삼역_Id;
+    private Long 선릉역_Id;
+    private Long 강남역_역삼역_거리 = 10L;
+    private Long 역삼역_선릉역_거리 = 15L;
+
+    @BeforeEach
+    void setup() {
+        super.setUp();
+
+        /**
+         * Given 지하철 노선을 등록하고
+         */
+        강남역_Id = 지하철역_등록됨("강남역").body().jsonPath().getLong("id");
+        역삼역_Id = 지하철역_등록됨("역삼역").body().jsonPath().getLong("id");
+        선릉역_Id = 지하철역_등록됨("선릉역").body().jsonPath().getLong("id");
+        lineId = 지하철_노선_생성됨("2호선", "green darken-2", 강남역_Id, 역삼역_Id, 강남역_역삼역_거리).jsonPath().getLong("id");
+    }
+
+    /**
+     * When 지하철 구간을 등록하면
+     * Then 지하철 구간 조회 시 생성한 구간을 찾을 수 있다.
+     */
+    @DisplayName("지하철 구간을 생성한다.")
+    @Test
+    void createSection() {
+        // when
+        지하철_구간_생성됨(lineId, new SectionRequest(선릉역_Id, 역삼역_Id, 역삼역_선릉역_거리));
+
+        // then
+        var response = 지하철_노선_Id로_조회됨(lineId).jsonPath();
+        assertThat(response.getList("stations.id")).contains(역삼역_Id.intValue(), 선릉역_Id.intValue());
```

Comment:
stations id가  Long type으로 내려오지 않아 intValue를 썼는데요
더 깔끔한 방법이 있을까요?

Reply:
```suggestion
        assertThat(response.getList("stations.id", Long.class)).contains(역삼역_Id, 선릉역_Id);
```
타입에 대한 파라미터를 추가하면 될 것 같습니다 :)